tutorials/classify_maizeland_NG

```diff
@@ -0,0 +1,234 @@
+---
```

Comment:
Please rename to `tutorials/classify-maizeland-ng/index.md` before requesting review.

Reply:
done...but the name change triggered some error message in the pull request check log; kindly confirm that this is inconsequential